Dockerfile

```diff
@@ -1,86 +1,381 @@
-FROM circleci/node:14
-RUN sudo npm install -g npm@7
-RUN npm -v
-WORKDIR /tmp
-RUN sudo apt-get update
-RUN sudo apt-get install -y \
-  xdg-utils \
-  libatk-bridge2.0-0 \
-  libgtk-3.0 \
-  libasound2 \
-  lsof \
-  sudo \
-  tcl \
-  expect \
-  zip \
-  lsof \
-  jq \
-  groff \
-  python \
-  python-pip \
-  libpython-dev \
-  less \
-  tree
-
-RUN sudo pip install awscli
+FROM public.ecr.aws/ubuntu/ubuntu:22.04 AS core
```

Comment:
this file is inspired by the ubuntu standard 6.0 build image that is provided by AWS Codebuild here:
https://github.com/aws/aws-codebuild-docker-images/tree/master/ubuntu/standard/6.0